packages/babel-preset-cli/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = () => ({
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: '8.0.0',
+        },
+      },
+    ],
+    '@babel/preset-typescript',
+  ],
+  plugins: ['@babel/plugin-proposal-class-properties', '@babel/plugin-proposal-object-rest-spread'],
```

Comment:
Node 8.6.0+ supports object rest & spread properties. Also class properties are part of TypeScript -- does the TS preset handle them?

Reply:
Good catch, removed the `@babel/plugin-proposal-object-rest-spread` transform.

`@babel/plugin-proposal-class-properties` is still needed – they're not transformed when only using `@babel/preset-typescript`.